Brooklyn/Contexts/Preferences/PreferencesWindowController.swift

```diff
@@ -73,6 +74,7 @@ private extension PreferencesWindowController {
     
     func setupLabels() {
         previewLabel.stringValue = manager.selectedAnimations.first?.name ?? ""
+        currentVersionLabel.stringValue = "Version: \(manager.currentVersion)"
```

Comment:
Can we move "Version" to a constant? Something like:

```swift
enum Strings {
    static let version = "Version"
}

We should migrate to `Localizable.strings` later and this should ease the migration

```

Reply:
Ah...sorry...missed that one in my first commit. I've added a new `Localizations` struct which wraps localizations. It is used like so:

`currentVersionButton.attributedTitle = NSAttributedString(string: "\(Localizations.Preferences.version) \(manager.currentVersion)")`

Not perfect but its the first steps to hide Localizations at least 😄

Reply:
Looks good to me! I'll probably create an issue to migrate to Localizable.strings for hacktoberfest. 

p.s.: Don't ask me why I didn't do it from the start, I guess I was more worried about solving actual problems ... 😓 🤦‍♂

Reply:
https://github.com/pedrommcarrasco/Brooklyn/issues/58 In case you want to give it a spin (**no pressure**)

Reply:
I'd like to give it a go sure, but it probably won't be till the weekend rolls by 😄

Reply:
Don't take me wrong @pbodsk, there's no pressure! It's something that will be done sooner or later, I just created the issue in case you want to use it for Hacktoberfest!

Reply:
No offence taken @pedrommcarrasco 😄. Seems that you've gotten some translations in French and Korean now, so it seems to have moved closer to sooner than later now huh 😄

Reply:
Yeah, that sooner or later is coming a lot sooner than I expected 😅 
Yeah, regarding Localizable strings, #60 is a step towards it. Then we just need to add more languages

Reply:
I can add Danish 🤷‍♂

Reply:
That would be great! 👍